### ISSUE TITLE
Lime support (compilation errors)

### ISSUE THREAD

**Issue by starburst997** [ncannasse/heaps#205]
I'm getting quite a lot of compilations errors when trying to build with lime.

One of them is "Type not found: [lime.graphics.opengl.FrameBuffer"](https://github.com/ncannasse/heaps/blob/master/h3d/impl/GlDriver.hx#L23). 
Which should be [GLFrameBuffer](https://github.com/openfl/lime/blob/develop/lime/graphics/opengl/GLFramebuffer.hx) I guess?

I'm going to post more and also try to debug when I'm going to have some time.

Perhaps I'm not using the correct lime version?

**Comment by ncannasse** [comment]
This might depend on the lime version, which one are you using ?

**Comment by starburst997** [comment]
I was using the latest Lime (3,5,0).

So I was able to get lime compilation working, these two PR are needed, [Heaps](https://github.com/ncannasse/heaps/pull/206) / [Lime](https://github.com/openfl/lime/pull/864).

Not sure if the PR for Lime should be added to Lime or if the modifications should be added to Heaps itself.

Also, the latest version from GIT at this time of Haxe and HXCPP is required (see this [issue](https://github.com/HaxeFoundation/hxcpp/issues/545)).

I just did some test under Linux, base2D / base3D worked very well. Only issue I had was when switching focus between windows, the app would sometime dies with segmentation fault. Did not have the time to dig much into it, but I guess this is for another issue.

Sound sample would not output any sounds on my machine and would hang with 100% cpu. Needed to convert the MP3 to OGG (since there is no MP3 support yet) and use stb_ogg_sound lib (with motion_twin's [patch](https://github.com/motion-twin/haxe_stb_ogg_sound)) to get compilation working.
